app/app.go

```diff
@@ -655,6 +644,26 @@ func New(
 		app.BankKeeper,
 	)
 
+	app.TransferInjectKeeper = transferinjectkeeper.NewTransferInject(
+		appCodec,
+		app.RollappKeeper,
+		app.BankKeeper,
+	)
+
+	// Create Transfer Keepers
+	app.TransferKeeper = ibctransferkeeper.NewKeeper(
+		appCodec,
+		keys[ibctransfertypes.StoreKey],
+		app.GetSubspace(ibctransfertypes.ModuleName),
+		app.TransferInjectKeeper,
```

Comment:
the ICS4Wrapper should be left `app.IBCKeeper.ChannelKeeper` 
as this is the layer that provides the writing capability for packets and acks